services/tasks/service.go

```diff
@@ -567,10 +567,7 @@ func (s *Service) writeContent(ctx context.Context, mediaType, ref string, r io.
 	if err != nil {
 		return nil, err
 	}
-	labels := map[string]string{
-		"containerd.io/gc.root": time.Now().UTC().Format(time.RFC3339),
-	}
-	if err := writer.Commit(ctx, 0, "", content.WithLabels(labels)); err != nil {
+	if err := writer.Commit(ctx, 0, ""); err != nil {
```

Comment:
You gotta leave it temporary here I think, otherwise, the gc could technically remove it before you create the image.

/cc @dmcgowan to confirm that behavior

Reply:
ya, the tests are failing because of that commit.  trying to figureout what I need to do really to handle it